app/src/main/java/co/touchlab/droidconandroid/shared/interactors/RsvpInteractor.java

```diff
@@ -2,23 +2,29 @@
 
 import android.support.annotation.NonNull;
 
+import com.birbit.android.jobqueue.JobManager;
+
 import java.util.UUID;
 
+import co.touchlab.droidconandroid.DroidconApplication;
 import co.touchlab.droidconandroid.shared.data.DatabaseHelper;
 import co.touchlab.droidconandroid.shared.data.Event;
+import co.touchlab.droidconandroid.shared.tasks.persisted.RsvpJob;
 import co.touchlab.squeaky.dao.Dao;
 import io.reactivex.Single;
 
 /**
  * Created by kgalligan on 4/7/16.
  */
 public class RsvpInteractor {
-    private final Long eventId;
+    private final Long           eventId;
     private final DatabaseHelper helper;
+    private final JobManager     jobManager;
 
     public RsvpInteractor(DatabaseHelper helper, Long eventId) {
         this.eventId = eventId;
         this.helper = helper;
+        this.jobManager = DroidconApplication.getInstance().getJobManager();
```

Comment:
Should pass jobManager in as a parameter to keep the application out of the interactors for doppl + eventual daggerism.